content/es/guides/directory-structure/nuxt.md

```diff
@@ -0,0 +1,76 @@
+---
+title: .nuxt
+description: El directorio de `.nuxt` se le llama *el directorio de construcción* (build directory). Es dinámicamente generado y escondido por defecto. Dentro de este directorio encontrarás archivos generados automaticamente cuando usas `nuxt dev` o los artefactos del proyecto cuando usas `nuxt build`.
+position: 1
+category: directory-structure
+questions:
+  - question: Qué comando generan el directorio .nuxt?
+    answers:
+      - nuxt start
+      - nuxt generate
+      - nuxt build or nuxt dev
+    correctAnswer: nuxt build or nuxt dev
+  - question: Qué propiedad usas para renombrar el archivo nuxt?
+    answers:
+      - dir
+      - build
+      - buildDir
+    correctAnswer: buildDir
+  - question: En que archivo puedes encontrar las rutas generadas?
+    answers:
+      - pages.js
+      - router.js
+      - views.js
+    correctAnswer: router.js
+  - question: Que puedes encontrar en el directorio de componentes?
+    answers:
+      - nuxt components
+      - custom components
+      - global components
+    correctAnswer: nuxt components
+  - question: Necesitas el directorio de .nuxt lo necesitas para publicar websites estáticos.
+    answers:
+      - cierto
+      - falso
+    correctAnswer: falso
+---
+
+El directorio de `.nuxt` se le llama _el directorio de construcción_ (_build directory_). Es dinámicamente generado y escondido por defecto. Dentro de este directorio encontrarás archivos generados automaticamente cuando usas `nuxt dev` o los artefactos del proyecto cuando usas `nuxt build`.
+
+<base-alert>
+
+El directorio `.nuxt` no deberia ser añadido en tu sistema de control de versión, y debería ser ignorado con `.gitignore`, ya que es generado automáticamente cuando ejecutas `nuxt dev` or `nuxt build`.
+
+</base-alert>
+
+### La propiedad de buildDir
+
+Por defecto, muchas herramientas web, asumen que `.nuxt` es un directorio escondido, esto se debe a que el nombre del archivo comiensa con un punto. Tu puedes usar la opción del _buildDir_ para prevenir eso. Si llegas a cambiar el nomrbe, recuerda añadir el nombre nuevo al archivo `.gitignore`
```

Comment:
The word **comiensa** is miswritten it should be written with a _z_ instead of an _s_. **_comienza_**. Also the word **nombre**  has a typo (nomrbe). 
Btw, I don't mean to be a grammar nazi or anything like that, sorry if it looks like it :P ,  I just think that proper written documentation in any project makes it look more professional and comitted.

Reply:
Don't worry man, I agree with you. I appreciate you going through it, I've been doing this while doing a website launch and was kind of rushed lol. So its nice that you took the time to check everything. I will push the pages.md and plugins.md tomorrow morning so will appreciate a look over those two as well.

Reply:
- [ ] Fix

Reply:
You are welcome and definitely will look over those tomorrow!
Hope that your launch went smooth!